.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - 10
```

Comment:
Added .travis.yml, so now you can integrate it with TravisCI and it will run the whole set of tests with every PR.